src/GregClient/Requests/PackageUpload.cs

```diff
@@ -36,6 +36,11 @@ public override Method HttpMethod
             get { return Method.POST; }
         }
 
+        internal override IList<Parameter> GetParamsToSign(ref RestRequest request)
+        {
+            return new List<Parameter>();// Nothig to sign yet.
```

Comment:
Typo: Nothig

Reply:
I'm wondering if it would be simpler to just have this function update `request` in place rather than returning a list.